### PR TITLE
Rename to wavefront-opentracing-sdk-python and add ApplicationTags

### PR DESCRIPTION
**Pull request by song-hao**

### CODE REVIEW

**Comment by ustinov** [comment]
Please run `pylint` against your changes

```
$ pylint wavefront_opentracing_sdk/
************* Module wavefront_opentracing_sdk.tracer
wavefront_opentracing_sdk/tracer.py:11:0: E0611: No name 'ApplicationTags' in module 'wavefront_sdk.common' (no-name-in-module)
wavefront_opentracing_sdk/tracer.py:11:0: W0611: Unused ApplicationTags imported from wavefront_sdk.common (unused-import)
************* Module wavefront_opentracing_sdk.propagation.http
wavefront_opentracing_sdk/propagation/http.py:12:4: W0107: Unnecessary pass statement (unnecessary-pass)
************* Module wavefront_opentracing_sdk.propagation.propagator
wavefront_opentracing_sdk/propagation/propagator.py:1:0: R0801: Similar lines in 2 files
==wavefront_opentracing_sdk.reporting.console:23
==wavefront_opentracing_sdk.reporting.wavefront:32
            wavefront_span.get_operation_name(),
            int(wavefront_span.get_start_time() * 1000),
            int(wavefront_span.get_duration_time() * 1000),
            self.source,
            wavefront_span.trace_id,
            wavefront_span.span_id,
            wavefront_span.get_parents(),
            wavefront_span.get_follows(),
            wavefront_span.get_tags(), (duplicate-code)
```

File: setup.py [hunk 1]
```diff
@@ -7,23 +7,23 @@
...
+VERSION = "1.0.0"
 # To install the library, run the following
 #
 # python setup.py install
 #
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ['opentracing>=2', 'wavefront_python_sdk']
+REQUIRES = ['opentracing>=2', 'wavefront-sdk-python']
```
Comment (ustinov) [inline setup.py]: Please specify version for `wavefront-sdk-python`

File: wavefront_opentracing_sdk/__init__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Wavefront Opentracing Python SDK.
+
+@author: Hao Song (songhao@vmware.com)
+"""
+
+from wavefront_opentracing_sdk.span_context import WavefrontSpanContext
+from wavefront_opentracing_sdk.span import WavefrontSpan
+from wavefront_opentracing_sdk.tracer import WavefrontTracer
+
+__all__ = ['WavefrontSpan', 'WavefrontSpanContext', 'WavefrontTracer',
```
Comment (ustinov) [inline wavefront_opentracing_sdk/__init__.py]: Redefining `__all__` here seems to be useless

File: wavefront_opentracing_sdk/tracer.py [hunk 3]
```diff
@@ -8,26 +8,30 @@
...
+from wavefront_opentracing_sdk.propagation import registry
+from wavefront_opentracing_sdk.span import WavefrontSpan
+from wavefront_opentracing_sdk.span_context import WavefrontSpanContext
 
 
 class WavefrontTracer(opentracing.Tracer):
     """Wavefront Tracer."""
 
-    def __init__(self, reporter, tags=None):
+    def __init__(self, reporter, application_tags, tags=None):
```
[the remark this answers is not in the document]
Comment (song-hao) [reply wavefront_opentracing_sdk/tracer.py]: I noticed this as well, will update tests accordingly

[the remark this answers is not in the document]
Comment (song-hao) [reply wavefront_opentracing_sdk/tracer.py]: Will also renaming `tags` to `global_tabs` to make it more clear

File: wavefront_opentracing_sdk/propagation/__init__.py [hunk 4]
```diff
@@ -4,10 +4,10 @@
...
-from wavefront_opentracing_python_sdk.propagation.textmap \
+from wavefront_opentracing_sdk.propagation.textmap \
     import TextMapPropagator
-from wavefront_opentracing_python_sdk.propagation.http import HTTPPropagator
-from wavefront_opentracing_python_sdk.propagation.registry \
+from wavefront_opentracing_sdk.propagation.http import HTTPPropagator
+from wavefront_opentracing_sdk.propagation.registry \
     import PropagatorRegistry
 
 __all__ = ['TextMapPropagator', 'HTTPPropagator', 'PropagatorRegistry']
```
Comment (ustinov) [inline wavefront_opentracing_sdk/propagation/__init__.py]: Redundant

File: wavefront_opentracing_sdk/reporting/__init__.py [hunk 5]
```diff
@@ -3,13 +3,13 @@
...
 
-from wavefront_opentracing_python_sdk.reporting.composite \
+from wavefront_opentracing_sdk.reporting.composite \
     import CompositeReporter
 
-from wavefront_opentracing_python_sdk.reporting.wavefront \
+from wavefront_opentracing_sdk.reporting.wavefront \
     import WavefrontSpanReporter
 
 __all__ = ['ConsoleReporter', 'CompositeReporter', 'WavefrontSpanReporter']
```
Comment (ustinov) [inline wavefront_opentracing_sdk/reporting/__init__.py]: Redundant

**Pull request merged**